libs/withdraws/src/lib/use-complete-withdraw.ts

```diff
@@ -55,7 +55,12 @@ export const useCompleteWithdraw = () => {
           approval.signatures
         );
       } catch (err) {
-        captureException(err);
+        const logger = localLoggerFactory({ application: 'deposits' });
```

Comment:
I don't think this helps solve the problem, it only moves the sentry error to a sentry message.

Instead lets try and capture more information on how we are calling the smart contracts. Then we can check that we are calling it with the right arguments (which is a likely culprit)

Lets do the following:

A) call logger.info with the supplied arguments
B) just call logger.error in the catch. We should be doing this more overall in the application so we aren't pulling Sentry directly into modules.

```
const submit = useCallback(() => {
  ...
  try {
    ...
    logger.info('calling withdraw_asset', ...args)
   } catch (err) {
     logger.error(err)
   }
 }, [])
```

Reply:
ok, I've pushed such a changes. Please take a look